foundry.toml

```diff
@@ -7,6 +7,4 @@ cache_path  = 'cache_forge'
 
 # 5667: Unused function parameter. Remove or comment out the variable name to silence this warning.
 # 5740: Unreachable code.
-ignored_error_codes = [5667, 5740]
-# setting this to allow expensive larger inserts during tests
-memory_limit = 100_000_000
```

Comment:
Any reason this was removed?

Reply:
yessir, the tests were grossly inefficient, forcing me to increase the memory more than the default. fixed the issue though, by mocking a revert!